HomeEdu/Scenes/Main/Login/LoginViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  LoginController.swift
+//  HomeEdu
+//
+//  Created by Nguyen Cuong on 7/31/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+
+final class LoginViewController: UIViewController {
+    
+    @IBOutlet private weak var userNameTextField: UITextField!
+    @IBOutlet private weak var passwordTextField: UITextField!
+    private var isLogin = false
+    private var isConnect = false
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        userNameTextField.text = ""
+        passwordTextField.text = ""
+    }
+    
+    @IBAction func buttonLogin(_ sender: Any) {
+        if checkValidInput() {
+            requestLogin()
+        }
+    }
+    
+    ///check input when user typing or click button
+    ///
+    /// - Returns: True: Input valid, False: Input invalid
+    func checkValidInput() -> Bool {
+        let username: String.validateType = .username
+        let password: String.validateType = .password
+        if let userText = userNameTextField.text, let passText = passwordTextField.text {
+            if !userText.isValidInput(username) &&
+                !passText.isValidInput(password) {
+                creatAlert(title: Vietnamese.Alert.alertTitle, message: Vietnamese.Alert.messageMissUserAndPass)
```

Comment:
```suggestion
                creatAlert(title: "alert.alertTitle", message: "(Ví dụ: alert.inValidPassword)")
```
Nguyên cứu https://medium.com/lean-localization/ios-localization-tutorial-938231f9f881
Còn nếu app chỉ support 1 ngôn ngữ thì thôi.